Automattic/pocket-casts-android
# Replace refresh section with Compose UI

## Description

This PR replaces refresh section with Compose UI.

## Testing Instructions

* Check refresh button in the profile section with different themes.
* Check that the button triggers the refresh.
* Disable the internet and tap on the button. An error icon should show. Tapping on it should show a dialog with the error message.
* Check the landscape orientation.

The button UI got updated to align with https://github.com/Automattic/pocket-casts-android/pull/3251

## Screenshots or Screencast 

### States

![Screenshot 2024-11-20 at 09 42 25](https://github.com/user-attachments/assets/b14bd6f3-ea59-4d41-ba13-a99c640f9b12)

### Theming

![Screenshot 2024-11-20 at 09 42 19](https://github.com/user-attachments/assets/42761328-dec5-44f7-aa6e-bbf2beca92e9)

## Checklist
 
#### I have tested any UI changes...
